linux/runtime.go

```diff
@@ -246,6 +248,14 @@ func (r *Runtime) Delete(ctx context.Context, c runtime.Task) (*runtime.Exit, er
 	if err := r.monitor.Stop(lc); err != nil {
 		return nil, err
 	}
+	bundle := loadBundle(
```

Comment:
Why does it need to be loaded earlier?

Reply:
because of the events before we kill the shim

Reply:
Sorry, I don't get it :/

The returned bundle is not referenced anywhere until we call `bundle.Delete()` and all `loadBundle()` does is creating a `&bundle{}` instance. What am I missing ?

Reply:
ya, you are right, the order does not matter here.